apps/web/src/services/asset-inventory/cloud-service/cloud-service-detail/modules/ExcelExportOptionModal.vue

```diff
@@ -16,17 +18,26 @@ import ErrorHandler from '@/common/composables/error/errorHandler';
 
 const props = defineProps<{
     visible: boolean;
+    cloudServiceId?: string;
+    isServerPage: boolean;
 }>();
 
 const emits = defineEmits<{(event: 'update:visible', value: boolean): void;
 }>();
 
+interface CloudServiceDetailSchema {
+    name: string;
+    type: string;
+    options: any;
+}
+
 const state = reactive({
     isValid: false,
     loading: false,
-    subDataList: ['Main Table', 'AWS EC2', 'Disk', 'NIC', 'Security Groups'] as string[], // TODO: dummy data
+    subDataList: computed(() => state.detailSchema.map((schema:CloudServiceDetailSchema) => schema.name)),
```

Comment:
suggetion: Please set type!🫡

Reply:
I will add the necessary type to the next PR as it is not clear in the API design yet!